### PR TITLE
Add dynamic wait for vm hibernate

### PR DESCRIPTION
**Pull request by shubham149**

### CODE REVIEW

File: internal/drivers/manager.go [hunk 1]
```diff
@@ -2,17 +2,19 @@ package drivers
...
 	"sort"
 	"sync"
 	"time"
 
+	"github.com/cenkalti/backoff/v4"
 	"github.com/drone-runners/drone-runner-aws/internal/certs"
 	"github.com/drone-runners/drone-runner-aws/store"
 	"github.com/drone-runners/drone-runner-aws/types"
 	"github.com/drone/runner-go/logger"
+	lehttp "github.com/harness/lite-engine/cli/client"
```
Comment (eoinmcafee00) [inline internal/drivers/manager.go:16]: import ordering

Comment (shubham149) [reply internal/drivers/manager.go:16]: I guess ordering is already correct. No changes are shown on go fmt

File: internal/drivers/manager.go [hunk 2]
```diff
@@ -2,17 +2,19 @@ package drivers
...
 import (
 	"context"
-	"errors"
 	"fmt"
 	"runtime/debug"
 	"sort"
 	"sync"
 	"time"
 
+	"github.com/cenkalti/backoff/v4"
```
Comment (eoinmcafee00) [inline internal/drivers/manager.go:11]: this should be below drone imports in a separate section. See how other files do it in the repo

Comment (eoinmcafee00) [reply internal/drivers/manager.go:11]: because it's 3rd party

Comment (eoinmcafee00) [reply internal/drivers/manager.go:11]: ignore you've already done it

**Review by eoinmcafee00 — APPROVED** [review]

**Pull request merged**